prusti-common/src/vir/ast/expr.rs

```diff
@@ -425,6 +430,13 @@ impl Expr {
         Expr::Cond(box guard, box left, box right, Position::default())
     }
 
+    pub fn int(value: i64) -> Self {
+        Expr::Const(
+            Const::Int(value),
+            Position::default(),
+        )
+    }
+
```

Comment:
We already have `Expr::from(value)`, which is implemented for many integer types.